North-Seattle-College/ad440-winter2022-thursday-repo
# Rearrange 'devops' folder

Takes care of #40 

**Description**

- `git `is case-insensitive by default when it comes to folder names and it appears we have two different devops folder in our github repo, as show below

![image](https://user-images.githubusercontent.com/26123463/153486188-495dec8b-de20-4a4b-872a-1f20f1680840.png)
So, to keep a consistent folder naming conventions I followed a fix [here ](https://stackoverflow.com/questions/18000138/duplicated-file-with-different-case-in-git-origin-bitbucket)


**To Test**

1. `git pull`
2. `git checkout dd-40`
3. Check the folder naming looks exactly as shown below

![image](https://user-images.githubusercontent.com/26123463/153485212-97ed3a05-bb98-412a-bc24-2aaa2c5dffe5.png)


4. Check the git repo folder structure looks exactly as show below


![image](https://user-images.githubusercontent.com/26123463/153485898-5f3c82c3-ef61-4ac9-8b05-ca5d4a8e677b.png)
